src/gps/gps/rtcm_pub_node.py

```diff
@@ -0,0 +1,239 @@
+import threading
+import rclpy
+import serial
+from rclpy.node import Node
+from rtcm_msgs.msg import Message as Rtcm
+from pyubx2.ubxtypes_configdb import SET_LAYER_RAM, SET_LAYER_BBR, SET_LAYER_FLASH
+from pyubx2 import (
+    RTCM3_PROTOCOL,
+    UBXMessage,
+    UBXMessageError,
+    UBXParseError,
+    UBXReader,
+    VALCKSUM,
+)
+
+# Defines
+TMODE_SVIN = 1
```

Comment:
We could make an enum class for this as we can expect to have more TMODE configuration options

Reply:
I do not expect to have any other TMODE options. Fixed and Survey in are the only two modes I am aware of?

Reply:
Misunderstanding on my part, yeah no the two should be fine.